### PR TITLE
Ep4: Explain echo command on first use

### LINKED ISSUE #877: echo command not explained on first use in pipes and filters
Please delete the text below before submitting your contribution. 

---
We have seen the use of >, but there is a similar operator >> which works slightly differently. By using the echo command to print strings, test the commands below to reveal the difference between the two operators:

Instead the echo command is explained in the next loops section. This explanation can be added where the command is first used.

The first, echo, just prints its command-line arguments to standard output. For example:
$ echo hello there
prints:
hello there
---

### PR DESCRIPTION
**Pull request by Murray2015**
Fixes #877

### CODE REVIEW

**Comment by gcapes** [comment]
Thanks for the PR!

This doesn't render correctly I'm afraid - you've forgotten to show the command which gives the output. I'll put some more comments on the diff. Could you please fix these before I merge the PR.

thanks

File: _episodes/04-pipefilter.md [hunk 1]
```diff
@@ -256,10 +256,14 @@ the output of `head` must be the file with the fewest lines.
...
 > the contents of `lengths.txt`.
 {: .callout}
-
 > ## What Does `>>` Mean?
 > We have seen the use of `>`, but there is a similar operator `>>` which works slightly differently.
-> By using the `echo` command to print strings, test the commands below to reveal the difference
+> First, let's try using the `echo` command to print strings, e.g.: 
+> ~~~
+> $ The echo command prints text
+> ~~~
```
Comment (gcapes) [inline _episodes/04-pipefilter.md]: This is a hybrid of the output and the command.
The command should be
```
$ echo The echo command prints text
```

and the corresponding output should be 
```
The echo command prints text
```

File: _episodes/04-pipefilter.md [hunk 2]
```diff
@@ -256,10 +256,14 @@ the output of `head` must be the file with the fewest lines.
...
+> {: .language-bash}
+> By using the `echo` command to print strings, let's now test the commands below to reveal the difference
```
Comment (gcapes) [inline _episodes/04-pipefilter.md]: I'm not sure how this is an improvement over the original wording in line 262?

File: _episodes/04-pipefilter.md (hunk 1 above)
Comment (Murray2015) [reply _episodes/04-pipefilter.md]: Yep, what a daft mistake. Well spotted.

Comment (gcapes) [reply _episodes/04-pipefilter.md]: Great, thanks. I think it would also be good to show the expected output in a code block.

**Comment by gcapes** [comment]
Thanks! 
Merged with a few edits in 98be3175b19d6b8fcac0a31f69c9193bdb4ff6c7

**Pull request closed without merging**